.rubocop_todo.yml

```diff
@@ -1,3 +1,134 @@
+Style/ExpandPathArguments:
+  Enabled: false
+Style/HashAsLastArrayItem: # (new in 0.88)
+  Enabled: false
+Style/HashLikeCase: # (new in 0.88)
+  Enabled: false
+Style/RedundantAssignment: # (new in 0.87)
+  Enabled: false
+Style/RedundantFetchBlock: # (new in 0.86)
+  Enabled: false
+Style/RedundantFileExtensionInRequire: # (new in 0.88)
+  Enabled: false
+Style/RedundantRegexpCharacterClass: # (new in 0.85)
+  Enabled: false
+Style/RedundantRegexpEscape: # (new in 0.85)
+  Enabled: false
+Style/SlicingWithRange: # (new in 0.83)
+  Enabled: false
+Performance/AncestorsInclude: # (new in 1.7)
+  Enabled: false
+Performance/BigDecimalWithNumericArgument: # (new in 1.7)
+  Enabled: false
+Performance/CollectionLiteralInLoop: # (new in 1.8)
+  Enabled: false
+Performance/DeletePrefix:
+  Enabled: false
+Performance/RedundantSortBlock: # (new in 1.7)
+  Enabled: false
+Performance/RedundantStringChars: # (new in 1.7)
+  Enabled: false
+Performance/ReverseFirst: # (new in 1.7)
+  Enabled: false
+Performance/SortReverse: # (new in 1.7)
+  Enabled: false
+Performance/Squeeze: # (new in 1.7)
+  Enabled: false
+Performance/StringInclude: # (new in 1.7)
+  Enabled: false
+Rails/ActiveRecordCallbacksOrder: # (new in 2.7)
+  Enabled: false
+Rails/ContentTag:
+  Enabled: false
+Rails/FindById: # (new in 2.7)
+  Enabled: false
+Rails/Inquiry: # (new in 2.7)
+  Enabled: false
+Rails/MailerName: # (new in 2.7)
+  Enabled: false
+Rails/MatchRoute: # (new in 2.7)
+  Enabled: false
+Rails/NegateInclude: # (new in 2.7)
+  Enabled: false
+Rails/Pluck: # (new in 2.7)
+  Enabled: false
+Rails/PluckInWhere: # (new in 2.7)
+  Enabled: false
+Rails/UniqueValidationWithoutIndex:
+  Enabled: false
+Rails/RedundantForeignKey:
+  Enabled: false
+Rails/IndexBy:
+  Enabled: false
+Rails/IndexWith: # (new in 2.8)
+  Enabled: false
+Rails/RenderInline: # (new in 2.7)
+  Enabled: false
+Rails/RenderPlainText: # (new in 2.7)
+  Enabled: false
+Rails/ShortI18n: # (new in 2.7)
+  Enabled: false
+Rails/WhereExists: # (new in 2.7)
+  Enabled: false
+Minitest/AssertInDelta: # (new in 0.10)
+  Enabled: false
+Minitest/AssertionInLifecycleHook: # (new in 0.10)
+  Enabled: false
+Minitest/AssertKindOf: # (new in 0.10)
+  Enabled: false
+Minitest/AssertOutput: # (new in 0.10)
+  Enabled: false
+Minitest/AssertPathExists: # (new in 0.10)
+  Enabled: false
+Minitest/AssertSilent: # (new in 0.10)
+  Enabled: false
+Minitest/GlobalExpectations:
+  Enabled: false
+Minitest/LiteralAsActualArgument: # (new in 0.10)
+  Enabled: false
+Minitest/MultipleAssertions: # (new in 0.10)
+  Enabled: false
+Minitest/RefuteInDelta: # (new in 0.10)
+  Enabled: false
+Minitest/RefuteKindOf: # (new in 0.10)
+  Enabled: false
+Minitest/RefutePathExists: # (new in 0.10)
+  Enabled: false
+Minitest/TestMethodName: # (new in 0.10)
+  Enabled: false
+Minitest/UnspecifiedException: # (new in 0.10)
+  Enabled: false
+
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 0`
```

Comment:
Can we continue using `rubocop --auto-gen-config --exclude-limit 0` to auto generate this file? If so, why were these additional cops added manually? ideally the todo file is autogenerated.